rpcserver.go

```diff
@@ -1156,11 +1157,18 @@ func (r *rpcServer) VerifyProof(ctx context.Context,
 	}
 	valid := err == nil
 
-	decodedProof, err := r.marshalProofFile(ctx, proofFile, 0, false, false)
+	p, err := proofFile.LastProof()
```

Comment:
Is this a bug fix? IIUC this is now handling the most recent proof instead of the genesis proof (which according to the godoc is what we want).

Reply:
Yes, I guess so. The parameter was called `depth` in `marshalProofFile`, even though it used that as the index. So before we would return the very first proof as the decoded proof. But the RPC comment mentions: `// The decoded last proof in the file if the proof file was valid.`, so I think it's correct now.